src/main/kotlin/apply/ui/api/ApplicantRestController.kt

```diff
@@ -20,34 +18,22 @@ class ApplicantRestController(
     private val mailService: MailService
 ) {
     @PostMapping("/register")
-    fun generateToken(@RequestBody applicantInformation: ApplicantInformation): ResponseEntity<String> {
-        return try {
-            val token = applicantService.generateToken(applicantInformation)
-            ResponseEntity.ok().body(token)
-        } catch (e: ApplicantValidateException) {
-            ResponseEntity.status(HttpStatus.UNAUTHORIZED).body(e.message)
-        }
+    fun generateToken(@RequestBody applicantInformation: ApplicantInformation): ResponseEntity<ApiResponse<String>> {
+        val token = applicantService.generateToken(applicantInformation)
+        return ResponseEntity.ok().body(ApiResponse(body = token))
```

Comment:
코드를 쓰기 쉽게 하는 건 어때요? 아래는 예시입니다.

```suggestion
        return ResponseEntity.ok().body(ApiResponse().body(token)))
```